package.json

```diff
@@ -8,22 +8,23 @@
     "babel-loader": "6.4.1",
     "babel-preset-es2015": "6.24.1",
     "babel-preset-react": "6.24.1",
+    "bootstrap": "4.0.0-alpha.6",
     "css-loader": "0.28.0",
+    "datepicker": "git+https://github.com/vellip/datePicker.git",
     "extract-text-webpack-plugin": "1.0.1",
     "file-loader": "0.11.1",
+    "font-awesome": "4.7.0",
+    "leaflet": "0.7.7",
+    "leaflet-draw": "0.4.9",
     "node-sass": "4.5.2",
     "postcss-loader": "1.3.3",
+    "react-flip-move": "2.9.2",
     "sass-loader": "4.1.1",
-    "style-loader": "0.16.1",
-    "webpack": "1.15.0",
-    "font-awesome": "4.7.0",
-    "bootstrap": "4.0.0-alpha.6",
     "sass-planifolia": "xi/sass-planifolia#0.3.1",
-    "react-flip-move": "2.9.2",
-    "tether": "1.4.0",
     "shariff": "1.24.1",
-    "leaflet": "0.7.7",
-    "leaflet-draw": "0.4.9"
+    "style-loader": "0.16.1",
+    "tether": "1.4.0",
+    "webpack": "1.15.0"
```

Comment:
at some point this was sorted into two categories: Build toolchain and actual runtime dependencies. `devDependencies` contains stuff like linters. I am not sure that this sorting is better.